File: Exercise/Easy/09. Sum of Two Numbers/Code/Sum of Two Numbers.cs
using System;

public class SumNumbers {
    public static void SumNumbersMethod(int num1, int num2) {
        Console.WriteLine(num1 + num2);
    }

    public static void Main(string[] args) {
        int num1 = int.Parse(Console.ReadLine());
        int num2 = int.Parse(Console.ReadLine());

        SumNumbersMethod(num1, num2);
    }
}

File: Exercise/Easy/09. Sum of Two Numbers/Code/Sum of Two Numbers.cpp
#include <iostream>

using namespace std;

void sumNumbers(int num1, int num2) {
    cout << num1 + num2 << endl;
}

int main() {
    int num1, num2;
    cin >> num1 >> num2;

    sumNumbers(num1, num2);

    return 0;
}